planetiler-core/src/main/java/com/onthegomap/planetiler/util/Translations.java

```diff
@@ -22,15 +22,31 @@ public class Translations {
     ThreadLocal.withInitial(() -> new ThreadLocalTransliterator().getInstance("Any-Latin"));
 
   private boolean shouldTransliterate = true;
-  private final Set<String> languageSet;
   private final List<TranslationProvider> providers = new ArrayList<>();
+  private final Set<String> includeLanguages;
+  private final Set<String> excludeLanguages;
+  private boolean defaultInclude = false;
 
   private Translations(List<String> languages) {
-    this.languageSet = new HashSet<>();
+    this.includeLanguages = new HashSet<>();
+    this.excludeLanguages = new HashSet<>();
+
     for (String language : languages) {
+      if (language.equals("*")) {
```

Comment:
Minor but what about supporting `"all"` instead of or in addition to `"*"` so that you don't need to worry about bash expanding it as a wildcard?

Reply:
Good idea, but `all` is a language code assigned to https://en.wikipedia.org/wiki/Allar_language. Any better ideas for a keyword?

Reply:
Ah, good catch! I can't think of any better short name than "all" so `"*"` seems good for now!